task/sast-unicode-check/0.1/README.md

```diff
@@ -0,0 +1,32 @@
+# sast-unicode-check task
+
+## Description:
+
+The sast-unicode-check task uses [find-unicode-control](https://github.com/siddhesh/find-unicode-control.git) tool to perform Static Application Security Testing (SAST) to look for non-printable unicode characters in all text files in a source tree.
+
+## Parameters:
+
+| name                         | description                                                                                                                                             |
+|------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
+| FIND_UNICODE_CONTROL_GIT_URL | URL from repository to find unicode control                                                                                                             |
+| KFP_GIT_URL                  | Known False Positives git URL, optionally taking a revision delimited by #; If empty, filtering of known false positives is disabled.                   |
+| PROJECT_NVR                  | Name-Version-Release (NVR) of the scanned project, used to find path exclusions (it is optional)                                                        |
+| RECORD_EXCLUDED              | Whether to record the excluded findings (it is optional, true or false). If `true`, the the excluded findings will be stored in `record_excluded.json`. |
```

Comment:
`it is optional, true or false` -> `defaults to false`

Please also change the file name to `excluded-findings.json` for consistency.

Reply:
Updated